src/DDTrace/Integrations/SandboxedIntegration.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace DDTrace\Integrations;
+
+use DDTrace\SpanData;
+use DDTrace\Tag;
+
+abstract class SandboxedIntegration extends Integration
+{
+    /**
+     * @var self
+     */
+    private static $instance;
```

Comment:
Unfortunately this won't work. Being a static instance the first sandboxed integration class will define the one and one instance. So I suggest to remove this instance thing and just call the constructor in `IntegrationLoader`. WDYT?